### ISSUE TITLE
What's the status of this project?

### ISSUE THREAD

**Issue by themattman18** [ellenfieldn/IdentityServer4.WsFederation#48]
What is the current status of this project? I see that identityserver guys archived the project and are now selling it instead of posting it on GitHub which lead me here. Is this project active and can you use it?

**Comment by ellenfieldn** [comment]
Hey, this project isn't particularly active.

I was thinking about putting some more work in here, but the rest of IdentityServer4 is scheduled to go EOL and they're packing their bags and doing a new project with a much more restrictive licensing model. (https://duendesoftware.com)

TBH, I'm just not sure how much of my time I want to put into maintaining the project at this point given that it's going to have to come to a dead-end.

TLDR: I probably wouldn't make any long-term plans to use this project unless you're willing to take on the burden of forking and maintaining all of IdentityServer to address any issues or security defects that come up in the future.
